Makefile

```diff
@@ -13,6 +13,7 @@ help:
 	@echo "make build                 - Build UserSpace CNI container."
 	@echo "make deploy                - Copy binary from container to host: /opt/cni/bin."
 	@echo "make all                   - build and deploy"
+	@echo "make rollout               - kubectl apply daemonset"
```

Comment:
I'm not a fan of calling it rollout, I would prefer make deploy for something like this...

can we rename to deploy and choose another name for the other command?
"make copy" ?

Reply:
agree